trackml/dataset.py

```diff
@@ -150,9 +153,10 @@ def list_prefixes(files):
                 yield x
 
 def _extract_event_id(prefix):
-    """Extract event_id from prefix, e.g. event_id=1 from `event000000001`.
+    """Extract event_id from prefix, e.g. event_id=1 from `event000000001`
+    or from `train_1/event000000001`
     """
-    return int(prefix[5:])
+    return int(prefix[prefix.find("event") + 5:])
```

Comment:
This logic trips up if directory contains the string `event`, which the train_sample.zip provided in challenge does. Otherwise seems to work fine. I added a fix here: https://github.com/LAL/trackml-library/pull/9